ibazel/ibazel.go

```diff
@@ -161,31 +167,33 @@ func (i *IBazel) Cleanup() {
 	}
 }
 
-func (i *IBazel) targetDecider(rule *blaze_query.Rule) {
```

Comment:
I commented on this in the PR comment. `targetDecider` may not be the best name since its only called right now for `run` targets and not for `build` or `test`. Hence the rename so its clear for listeners when to expect that hook. Maybe `runTargetDecider` is better?